wren-ui/src/apollo/server/resolvers/projectResolver.ts

```diff
@@ -131,10 +164,34 @@ export class ProjectResolver {
     ctx: IContext,
   ) {
     const { type, properties } = args.data;
-    await this.removeCurrentProject(ctx);
+    // Curretly only support one data source
+    await this.resetCurrentProject(_root, args, ctx);
```

Comment:
What does it mean "Currently only support one data source" ?

Reply:
I changed it to `Currently only can create one project`, is that ok?

Reply:
sure